components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Extensions/SecurityConfigEx.cs

```diff
@@ -50,7 +50,8 @@ public static class SecurityConfigEx {
                 AutoAcceptUntrustedCertificates = securityConfig.AutoAcceptUntrustedCertificates,
                 RejectSHA1SignedCertificates = securityConfig.RejectSha1SignedCertificates,
                 MinimumCertificateKeySize = securityConfig.MinimumCertificateKeySize,
-                ApplicationCertificate = securityConfig.ApplicationCertificate.ToCertificateIdentifier()
+                ApplicationCertificate = securityConfig.ApplicationCertificate.ToCertificateIdentifier(),
+                AddAppCertToTrustedStore = true
```

Comment:
>AddAppCertToTrustedStore [](start = 16, length = 24)

same as last comment in previous file, better make it false

Reply:
same as above